app/helpers/get_involved_helper.rb

```diff
@@ -0,0 +1,34 @@
+module GetInvolvedHelper
+  def date_microformat(attribute_name)
+    attribute_name.to_date.strftime("%d %B %Y")
+  end
+
+  # Gets the link to the search page for all consultations
+  def get_consultations_link(filters = %w[open_consultations closed_consultations])
+    "search/policy-papers-and-consultations?#{filters.to_query('content_store_document_type')}"
```

Comment:
Does this need to be relative (i.e. with a slash at the start), otherwise it could link to somewhere unintended?

Reply:
Oops